src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -123,6 +142,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
         array_pop($rows);
         $io->section('Loader Paths');
         $io->table(array('Namespace', 'Paths'), $rows);
+        $messages = $this->buildWarningMessages($this->findWrongBundleOverrides());
+        array_walk(
+            $messages,
+            function ($message) use ($io) {
+                $io->warning(trim($message));
```

Comment:
Can't we just trim in `buildWarningMessages()` and then iterate the array here?

Reply:
Done